source/connect-atlas-cli.txt

```diff
@@ -96,6 +96,16 @@ Complete the Prerequisites
 
       - :ref:`Install the {+atlas-cli+} <install-atlas-cli>`.
       - Add your host's IP address to the :ref:`IP access list <access-list>`.
+  
+        If you log into the {+atlas-cli+} with your Atlas user credentials,
+        and your organization's owners enable
+        :atlas:`IP access list for the Atlas UI for an organization
+        </tutorial/manage-organizations/#require-ip-access-list-for-the-atlas-ui>`,
+        then to run any commands in this organization, your IP address must
+        be included in the IP access list or CIDR ranges defined in that list.
```

Comment:
Suggestion: simplifies the text here ("must be added" could be either as a single IP address or CIDR range)
```suggestion
        your IP address must be added to the IP access list to run 
        commands in this organization.
```